web/rootfs/defaults/settings-config.js

```diff
@@ -364,3 +364,13 @@ config.disableReactions = {{ $DISABLE_REACTIONS }};
 
 // Polls
 config.disablePolls = {{ $DISABLE_POLLS }};
+
+// configure toolbar buttons
```

Comment:
Nit: can you properly capitalize `configure`?

Reply:
Sure